examples/huggingface/pytorch/text-to-image/deployment/stable_diffusion/README.md

```diff
@@ -76,11 +75,10 @@ By setting --bf16 to export FP32 and BF16 models.
 python prepare_model.py --input_model=CompVis/stable-diffusion-v1-4 --output_path=./model --bf16
 ```
 
-By setting --qat_int8 to export INT8 models, **only support runwayml/stable-diffusion-v1-5**.
-
-**NOTE**: You should get a fake_quantized_model_qinit.pt before prepare qat models. Please refer the [link](https://github.com/intel/intel-extension-for-transformers/blob/main/examples/huggingface/pytorch/text-to-image/quantization/qat/README.md).
+For INT8 quantized mode, we **only support runwayml/stable-diffusion-v1-5** for now.
+You need throught QAT to get a quantize int8 model at first, Please refer the [link](https://github.com/intel/intel-extension-for-transformers/blob/main/examples/huggingface/pytorch/text-to-image/quantization/qat/README.md).
```

Comment:
You need throught QAT to get a quantize int8 model at first => You need to get a quantized INT8 model first through QAT

Reply:
Fixed